airflow/.env.sample

```diff
@@ -1,2 +1 @@
-CALITP_AMPLITUDE_BENEFITS_API_KEY=api-key
-CALITP_AMPLITUDE_BENEFITS_SECRET_KEY=secret-key
+
```

Comment:
@evansiroky @lauriemerrell Pre-commit does not like that this file is now blank. Should I add a comment in here so it's not blank? Or is it okay to delete this file entirely?

Reply:
I think it's safe to delete this file entirely!

Reply:
Thanks @lauriemerrell - Resolved here: https://github.com/cal-itp/data-infra/pull/3185/commits/46a4757fa159bfebef5cb4cbffe6e4fc992e0045